test/jest/jest-text-transformer.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
```

Comment:
I hope you didn't write this from memory. 😅

Reply:
Heh, no, that came from https://github.com/janouma/jest-text-transformer.

Actually given that I copy-pasted it, I should attribute the source in here.